### ISSUE TITLE
Remove the PrivacySandbox Chromium feature

### ISSUE THREAD

**Issue by Sa-Ja-Di** [brave/brave-browser#30644]
Description=
Recently Google has added towards Chromium a feature called "Privacy Sandbox - which works in similar ways to Brave's ethical ads.

This feature can be reached over
brave://settings/privacySandbox

Expected result=
This feature should be removed from Brave

**Comment by Arandhras** [comment]
I agree wholeheartedly with what Sa-Ja-Di says. Not only this feature is reduntant with respect to Brave's "ethical ads", but it is a potential privacy attack towards the user since it enables a third-party site to track him/her while not using cookies.

The solution is twofold: either enable the user to disable the feature completely, or remove it altogether from Chromium's code -- which is what Microsoft Edge is doing.

**Comment by Ayern12** [comment]
##ok...Continue If it's not good, or malicious..

Pada Ahd, 28 Mei 2023, 19:30 Andrea Salvi ***@***.***>
menulis:

**Comment by Sa-Ja-Di** [comment]
> I agree wholeheartedly with what Sa-Ja-Di says. Not only this feature is reduntant with respect to Brave's "ethical ads", but it is a potential privacy attack towards the user since it enables a third-party site to track him/her while not using cookies.
> 
> The solution is twofold: either enable the user to disable the feature completely, or remove it altogether from Chromium's code -- which is what Microsoft Edge is doing.

I would vote for remove it completely - this anti-privacy feature should have no place in Brave at all - at the very least they should make that internal Url going nowhere, that would be at least a first good step.

**Comment by Sa-Ja-Di** [comment]
Actually there is a patch available on Ungoogled Chromium which seems to do that job.... no Privacy Sandbox present in there:

https://github.com/ungoogled-software/ungoogled-chromium/blob/master/patches/core/ungoogled-chromium/disable-privacy-sandbox.patch

**Comment by HLFH** [comment]
cc @BrendanEich. Has Brave any plans in regards to the Privacy Sandbox?
